dts/riscv/espressif/esp32h2/esp32h2_common.dtsi

```diff
@@ -207,5 +210,123 @@
 			interrupt-parent = <&intc>;
 			clocks = <&clock ESP32_UART1_MODULE>;
 		};
+
+		usb_serial: uart@6000f000 {
+			compatible = "espressif,esp32-usb-serial";
+			reg = <0x6000F000 0x1000>;
```

Comment:
would you mind defaulting the reg size for all entries either to hex or using `DT_SIZE_K`? I see there is mixed usage in here.